uwblueprint/focus-on-nature
# Registrant Experience: Review Information Step (Additional Info)

## Notion ticket link

[Registrant Experience: Review Information Step (Additional Info)](https://www.notion.so/uwblueprintexecs/Registrant-Experience-Review-Information-Step-Additional-Info-d723a8b7740847c7be9ede62c695a60b)


## Implementation description
* Added an `EditAdditionalQuestionsCard.tsx` component, very similar to the other edit components in the same folder (editing the camper and contact card)
* Also added an edit header (`EditCardHeader.tsx`) to abstract away some of the common implementations
* Made lots of design updates mostly regarding responsiveness to the additional info step (step 2 of registration), which was needed since my component uses the same styles



## Steps to test
1. Run it locally, try to register any camp (i.e. http://localhost:3000/register/camp/63139c7bc3d7b55b44a01531) and ensure that on the review page, everything works as required
2. On all steps of the registration (notably the second step), resize the window and test responsiveness of all components.


https://user-images.githubusercontent.com/30396273/212221152-f94ddea6-28a8-44ab-b69c-58b005589cc4.mp4



## What should reviewers focus on?
* Responsiveness and attention to design decisions (i.e. on Figma)
* Functionality


## Checklist
